ci-operator/step-registry/syndesisio/deploy-syndesis-qe/syndesisio-deploy-syndesis-qe-ref.yaml

```
@@ -0,0 +1,20 @@
+ref:
+  as: syndesisio-deploy-syndesis-qe
+  from: cli
```

```suggestion
  from: fuse-online-test-runner
```

If this is the image holding the environment that should be used for PQE test script that you call in `syndesisio-deploy-syndesis-qe-commands.sh` then you should put this here instead of cli